### DISCUSSION TITLE
C109 value

### DISCUSSION THREAD

**Discussion by yogiVan** [frntc/SIDKick#2] [Q&A]
Thanks for this great project. Not sure about the value of C109. listed as 'x5r g1206 100/6'  so is this a 1206 size 100uF 6v x5r? Mouser drew a blank search for the BOM listing.

**Comment by frntc** [comment]
yes -- in my test builds I used various 100uF x5r capacitors. Voltage should be >= 6V (I mostly used 10V), size is 1206. Mouser/Digikey/LCSC all have the GRM31CR61A107MEA8L which works well.

**Comment by yogiVan** [comment]
Thanks for the info. Glad to know, I had ordered just about the same Murata cap(GRM31CR60J107MEA8K) which is a 6.5v 20% tol MLCC.Again thank you for this project, so cool.


William E Van Buskirk Cell 301-789-3467 Home 301-860-0844 ***@***.*** 

    On Sunday, October 8, 2023 at 06:12:12 AM EDT, frntc ***@***.***> wrote:  
 
 


yes -- in my test builds I used various 100uF x5r capacitors. Voltage should be >= 6V (I mostly used 10V), size is 1206. Mouser/Digikey/LCSC all have the GRM31CR61A107MEA8L which works well.
